README.md

```diff
@@ -28,6 +28,8 @@ Install-Module AzureAd
 # Deployment
 
 To deploy the sample scenario, first clone this git repo and find the deployment scripts folder:
+**NOTE** You must use Windows Powershell, not Powershell Core! 
+**NOTE** AzureRM will cause a conflict in some cases. You can use ```Powershell Uninstall-AzureRM``` cmdlet to eliminate possible conflicts with differing Azure modules. 
```

Comment:
That is not really related to this repo. `AzureRm` is being deprecated, and there are plenty of instructions elsewhere for switching.